dom/nodes/insertion-effects/Node-appendChild-text-and-script-in-style.tentative.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting text and script nodes in a style element</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style id="style"></style>
+<body>
+<script>
+const happened = []
+const style = document.getElementById("style");
+test(() => {
+  const r1 = new Text("body {}");
+  const r2 = new Text("body {}");
+  const script = document.createElement("script");
+  script.textContent = `
+    happened.push(style.sheet.cssRules.length);
```

Comment:
This is related to my comment in the file `dom/nodes/insertion-effects/Node-appendChild-script-and-style-from-fragment.tentative.html`. It seems like the expectations in this file (and that one, mostly) expect "update the style" synchronously in the middle of insertion, whereas script executes *after*, and can observe all script-observable changes that styles contribute, regardless of insertion order. I've added a test description below describing that. LMK if it matches your understanding.

Reply:
Fixed the style tests

Reply:
Thanks!